scripts/dvm

```diff
@@ -363,6 +368,39 @@ dvm_implode() {
   fi
 }
 
+# Pure bash version comparison
+# returns 0 if $1 >= $2
+vercomp () {
```

Comment:
Thanks for all these suggestions, I'll get it all cleaned up so this can get merged in. Also, although older Dartiums were time-bombed, I'd note that the DARTIUM_EXPIRATION_TIME env variable exists to allow dartium builds to be used past their sell-by date.